src/components/App.js

```diff
@@ -57,11 +58,30 @@ export default class App extends PureComponent {
 
 	componentWillUnmount() {
 		cliCursor.show(this.props.stdout);
-		this.handleSetRawMode(false);
+
+		// ignore calling setRawMode on an handle stdin it cannot be called
+		if (this.isRawModeSupported(this.props.stdin)) {
+			this.handleSetRawMode(false);
+		}
+	}
+
+	isRawModeSupported(stdin) {
+		return stdin.isTTY;
```

Comment:
Is there a reason this method accepts `stdin`, instead of reading it directly from `this.props.stdin`?

Reply:
My thinking was that it's more natural to not read from this.props directly, which makes it more clear what exactly that function does. It might not even have to be an instance method on App